corehq/apps/smsbillables/generator.py

```diff
@@ -150,7 +151,12 @@ def arbitrary_fees_by_all(backend_ids):
 def arbitrary_backend_ids():
     backend_ids = {}
     for backend in get_available_backends().values():
-        backend_ids[backend.get_api_id()] = data_gen.arbitrary_unique_name("back")
+        backend_instance = data_gen.arbitrary_unique_name("back")
+        backend_ids[backend.get_api_id()] = backend_instance
+        sms_backend = SMSBackend()
```

Comment:
We don't need to send SMSs in the smsbillables tests, so they don't need to be subclassed.  It's only used to test the is_global logic added in this PR